Makefile

```diff
@@ -323,10 +322,9 @@ lint-fix:
 ##                                 Protobuf                                  ##
 ###############################################################################
 
-DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf:1.8.0
-
-protoVer=v0.7
-protoImageName=tendermintdev/sdk-proto-gen:$(protoVer)
+protoVer=0.13.0
```

Comment:
let's use v0.7 and then we can merge this PR

Reply:
there is no v0.7 for https://github.com/cosmos/cosmos-sdk/pkgs/container/proto-builder/versions?filters%5Bversion_type%5D=tagged